docs/key_features.md

```diff
@@ -5,16 +5,19 @@
 - Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
   - gridded Earth system data (e.g., numerical weather prediction models)
```

Comment:
```suggestion
- Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the [dimensionality](project:./tutorials/Dimension_Handling.md), data size and data structures commonly used for:
  - gridded Earth system data (e.g., numerical weather prediction models)
```
I was wondering if we should hyperlink the "Handling dimensions with scores" notebook here?

Reply:
I think that's a nice suggestion